notebooks/triposr-3d-reconstruction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    HF_TOKEN = os.getenv("HF_TOKEN")
</pre><p>does it required some specific access rights? Why do you need explicitly provide hf token?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>removed, it's not required</p>